packages/actor-extract-links-solid-type-index/lib/ActorExtractLinksSolidTypeIndex.ts

```diff
@@ -144,6 +145,54 @@ export class ActorExtractLinksSolidTypeIndex extends ActorExtractLinks {
     return typeLinks;
   }
 
+  /**
+   * To fetch the domain of the predicate.
+   * @param predicateSubjects A dictionary of predicate and its subjects from the query.
+   * @param typeSubjects A dictionary of class type and its subjects from the query.
+   */
+  public async linkPredicateDomain(predicateSubjects: Record<string, RDF.Term>,
```

Comment:
```suggestion
  public async linkPredicateDomains(predicateSubjects: Record<string, RDF.Term>,
```
(To remain consistent with the method below)